.github/workflows/unit-test.yml

```diff
@@ -3,23 +3,6 @@ name: "Unit Test"
 on: [ push,pull_request ]
 
 jobs:
-  go:
-    name: Test go and binlog generation
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set up Go 1.x
-        uses: actions/setup-go@v2
-        with:
-          go-version: "1.20"
-      - name: Install protoc
-        uses: arduino/setup-protoc@v1
-        with:
-          version: '3.5.1'
-      - name: Checkout repository
-        uses: actions/checkout@v2
-      - name: Test
-        run: make go binlog && git diff --exit-code
```

Comment:
Sorry I see there's `make go` to ensure golang protobuf generation. So this part should be changed to

```
  go:
    name: Test go generation
    runs-on: ubuntu-latest
    steps:
      - name: Set up Go 1.x
        uses: actions/setup-go@v2
        with:
          go-version: "1.20"
      - name: Install protoc
        uses: arduino/setup-protoc@v1
        with:
          version: '3.5.1'
      - name: Checkout repository
        uses: actions/checkout@v2
      - name: Test
        run: make go && git diff --exit-code
```